app/views/details/info.html.erb

```diff
@@ -0,0 +1,124 @@
+            <span></span>
+            <div id="confirmMessage"></div>
+            <div class="space-10"></div>
+            <div class="title-bar blue"></div>
+          </div>
+        </div>
+        <div class="row relative-position fix">
+          <div class="col-xs-12 col-md-6 book-list-position padding60">
+            <div class="book-list-photo">
+              <div class="book-list owl-carousel owl-theme owl-center owl-loaded">
+                <div class="owl-stage-outer">
+                  <div class="owl-stage">
+                    <div class="book-item" id="info_image">
+                      <%= link_to image_tag("happiness-now.jpg", alt: "list book"), 
+                        "app/assets/images/happiness-now.jpg" %>
+                    </div>          
+                  </div>
+                </div>
+                <div class="owl-controls">
+                  <div class="owl-nav">
+                  </div>
+                  <div class="owl-dots">
+                    <div class="owl-dot active">
+                      <span></span>
+                    </div>
+                    <div class="owl-dot">
+                      <span></span>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-xs-12 pull-right col-md-6 padding60 wow fadeInRight">
+            <div class="book-content">
+              <div class="book-details-item">
+                <div class="row">
+                  <p></p>
+                  <h4 class="tip-left">
+                    <%= t ".author" %>
+                    <span class="text-warning"><%= t ".andrew" %></span>
+                  </h4>
+                  <p></p>
+                  <div class="rating-container rating-xs rating-animate rating-disable"></div>
+                </div>
+                <div class="space-10"></div>
+                <div class="row">
+                  <p></p>
+                  <h4 class="tip-left">
+                    <%= t ".category" %>
+                    <span class="text-warning"><%= t ".tamly" %></span>
+                  </h4>
+                  <p></p>
+                </div>
+                <div class="space-10"></div>
+                <div class="row">
+                  <h4 class="tip-left">
+                    <%= t ".shared_by:" %>
+                  </h4>
+                    <div class="text-warning lead"><%= t ".name1" %></div>
+                </div>
+                <div class="space-10"></div>
+                <div class="row">
+                  <h4 class="tip-left">
+                    <%= t ".branch" %>              
+                  </h4>
+                  <div class="text-warning lead"><%= t ".office" %></div>
+                </div>
+                <div class="row">
+                  <div class="check-status">
+                    <h3><%= t ".status" %></h3>
+                    <%= form_for :a, url: "#" do |f| %>
```

Comment:
What is :a here ? What is the purpose ? Can you explain in brief ?

Reply:
it's "form_for :status". I will create "status model" and use "form_for :status" to create or update status of a book user